Test/model_test.ts

```
@@ -265,4 +265,61 @@ describe('model methods', () => {
      * - user chains with incompatible methods (i.e. not after 'select' method)
      */
   });
+
+  describe('order method', () => {
+    it('adds appropriate query string to model when order is invoked with ASC', () => {
+      Planet['sql'] = 'SELECT * FROM planets';
+      const actualQuery =
+        Planet.order('ASC', 'diameter', 'rotation_period')['sql'];
+      assert(Planet['sql'].includes('ORDER BY diameter,rotation_period ASC'));
+    });
+
+    it('adds appropriate query string to model when order is invoked with DESC', () => {
+      Planet['sql'] = 'SELECT * FROM planets';
+      const actualQuery =
+        Planet.order('DESC', 'diameter', 'rotation_period')['sql'];
+      assert(Planet['sql'].includes('ORDER BY diameter,rotation_period DESC'));
+    });
+
+    /**
+     * no tests for the following problems (for which Postgres will throw its own errors):
+     * - user invokes 'order' method without an argument
+     * - user invokes without either ASC or DESC
+     * - user invokes with column or table names not in the database
```

add test for seeing if an error gets thrown for input other than ASC or DESC.
It should also throw an error for the column not matching and incorrect chaining.